packages/fleet/lib/supervisor/supervisor.ts

```diff
@@ -109,7 +109,7 @@ export class Supervisor {
         while (this.state === 'running') {
             const res = await withPgLock({
                 db: this.dbClient.db,
-                lockKey: `fleet_supervisor`,
+                lockKey: envs.FLEET_SUPERVISOR_LOCK_KEY,
```

Comment:
I would use the fleetId instead of another env var. `lockKey: 'fleet_superviser_${this.fleetId}'`

I would also use fleetId for the other lock. https://github.com/NangoHQ/nango/blob/4e2c1d4e4740334853b2867af3609f00b03118ec/packages/fleet/lib/fleet.ts#L113
The routing id might gonna be different but just in case

Reply:
Was looking for the most backward compatible approach.

I was worried about changing the lock key during a deployment to production - will that not cause any issues? Happy to make this change if that's not a problem.

Reply:
good point